tests/lib/Validator/Constraint/PasswordTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformUser\Tests\Validator\Constraint;
+
+
+use EzSystems\EzPlatformUser\Validator\Constraints\Password;
+use EzSystems\EzPlatformUser\Validator\Constraints\PasswordValidator;
+use PHPUnit\Framework\TestCase;
+
+class PasswordTest extends TestCase
+{
+    /** @var \EzSystems\EzPlatformUser\Validator\Constraints\Password */
+    private $constraint;
+
+    protected function setUp(): void
+    {
+        $this->constraint = new Password();
+    }
+
+    public function testConstruct()
```

Comment:
Could you use `: void` for test methods as well?
```suggestion
    public function testConstruct(): void
```